<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1514</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Automated Particle Segmentation [IMWS]</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Image</string>
    </property>
    <widget class="QPushButton" name="pb_loadImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_deleteImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Image</string>
     </property>
    </widget>
   </widget>
   <widget class="GraphicsLayoutWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>961</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>160</y>
      <width>471</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>451</width>
       <height>491</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Segmentation</string>
      </attribute>
      <widget class="QComboBox" name="cb_lib">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="currentText">
        <string>cuda</string>
       </property>
       <item>
        <property name="text">
         <string>cuda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cpu</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pb_segment">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>320</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_pps">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>310</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>340</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>points_per_side:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of points to be sampled along one side of&lt;br/&gt;the image. The total number of points is points_per_side**2.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>min_mask_region_area:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_pps_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If &amp;gt;0, postprocessing will be applied to remove disconnected regions&lt;br/&gt;and holes in masks with area smaller than min_mask_region_area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>411</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If &amp;gt;0, mask prediction will be run again on crops of the image. Sets&lt;br/&gt;the number of layers to run, where each layer has 2**i_layer number&lt;br/&gt;of image crops&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Preprocessing</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Plot result</string>
      </attribute>
      <widget class="QPushButton" name="pb_hist">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot histogram</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Pixel size</string>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Âµm</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1514</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuClose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
